src/frontend/komponenter/Fagsak/Vilkår/BehandleVilkårSkjema.tsx

```diff
@@ -82,6 +82,21 @@ const BehandlingVilkårSkjema: React.FunctionComponent<IBehandlingVilkårSkjema>
                     });
                 }}
             />
+
+            <br/>
+            
+            <TextareaControlled 
+                label={'Begrunnelse'}
+                maxLength={0}
+                defaultValue={context.begrunnelse}
+                onBlur={(evt:any)=>{
+                    dispatch({
+                        type: actions.SETT_BEGRUNNELSE,
+                        payload: evt.target.value,
+                    })
+                }}
+            />
```

Comment:
We need some kind of validation here. It is required at least, but not limited to characters.

Reply:
should we use IFelt?

Reply:
Yes! 👍